lib/ruby_lsp/requests/code_actions.rb

```diff
@@ -48,6 +48,8 @@ def initialize(document, range, context)
 
       sig { override.returns(T.nilable(T.all(T::Array[Interface::CodeAction], Object))) }
       def perform
+        return if @document.is_a?(ERBDocument)
```

Comment:
Perhaps these checks could be something like `@document.supports_code_actions?`, to ease the introduction of other document types in future.